src/server/index.js

```diff
@@ -57,7 +57,10 @@ export const listen = async ({
       port,
     });
 
-    console.log('%s listening on port %d', context, port);
+    console.info('%s listening on port %d', context, port);
+    if (context === '🌎 One App server') {
+      console.log('🚀 One App Server is running on http://localhost:%d 🚀', port);
```

Comment:
This URL is only accurate in development, but will be logged in production

Reply:
I think we should actually add a new log level for cases like this. `console.dev`

It would require a change here:

https://github.com/pinojs/pino-abstract-transport/pull/90/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R67-R71

and here

https://github.com/americanexpress/one-app/blob/main/src/server/utils/logging/monkeyPatchConsole.js#L19

```
  customLevels: {
    dev: process.env.NODE_ENV === 'development' ? 34 : 0,
    log: 35,
  },
```

This would allow us to have logs that are only for development, but not debugging, without carrying around this condition

Reply:
For reference in the above, `console.info` is `30` and `console.warn` is `40`

Reply:
@giulianok that kind of defeats the purpose of the level. the existing log levels (trace, debug, info, log, warn, error, fatal) all have practical cases in both development and productions. these would strictly be for development.

If the point is about using console though, @PixnBits also pointed out and I agreed we should replace out use of `console[method]` with importing the logger and using `logger[method]`. IIRC everything left in src that uses `console[method]` is outside the fastify context. Anything within that context should be using `fastify.log[method]` or `req.log[method]` (if in the context of a request)